spotipy/util.py

```diff
@@ -35,21 +35,28 @@ def prompt_for_user_token(
         "    spotipy.Spotify(auth_manager=auth_manager)",
         DeprecationWarning
     )
-    """ prompts the user to login if necessary and returns
-        the user token suitable for use with the spotipy.Spotify
-        constructor
-
-        Parameters:
-
-         - username - the Spotify username (optional)
-         - scope - the desired scope of the request (optional)
-         - client_id - the client id of your app (required)
-         - client_secret - the client secret of your app (required)
-         - redirect_uri - the redirect URI of your app (required)
-         - cache_path - path to location to save tokens (optional)
-         - oauth_manager - Oauth manager object (optional)
-         - show_dialog - If true, a login prompt always shows (optional, defaults to False)
-
+    """Prompt the user to login if necessary and return a user token.
+
+        Prompts the user to login if necessary and returns the user token
+        suitable for use with the `spotipy.Spotify` constructor.
```

Comment:
Can you explain why you've decided to duplicate the first part of the sentence?

Reply:
I was making this change in a closed PR but due to commit and repo issues, I updated it to this request and mistakenly left the sentence as I was making the updated changes to this one. I've corrected it.